content/blog/2021-08-24-pre-defined-models-exp.md

```diff
@@ -0,0 +1,374 @@
+```
+
+You'll see a table similar to this.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ neutral:**Created**  ┃ metric:**step** ┃     metric:**acc** ┃    metric:**loss** ┃ metric:**training_time** ┃ param:**lr**   ┃ param:**momentum** ┃ param:**model_name** ┃ param:**num_classes** ┃ param:**batch_size** ┃ param:**num_epochs** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━┩
+│ **workspace**               │ -        │    **4** │ **0.91803** │  **33.434** │        **225.35** │ **0.9**  │ **0.35**     │ **alexnet**    │ 2           │ 8          │ 5          │
+│ **main**                    │ **10:27 AM** │    **1** │ **0.88525** │ **0.28284** │        **82.143** │ **0.01** │ **0.09**     │ **squeezenet** │ **2**           │ **8**          │ **2**          │
+│ │ ╓ a9b7e7c [exp-2e1e1] │ 10:33 AM │    4 │ 0.91803 │  33.434 │        225.35 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 71d1e30             │ 10:32 AM │    4 │ 0.91803 │  33.434 │        225.35 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 18c6b09             │ 10:31 AM │    3 │ 0.92623 │  23.869 │        175.51 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ bb2154d             │ 10:31 AM │    2 │ 0.84426 │  52.716 │        125.56 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 4e2f7c4             │ 10:30 AM │    1 │ 0.86066 │  67.566 │        75.783 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ ├─╨ 6ad572d             │ 10:29 AM │    0 │    0.75 │  102.48 │         25.17 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+└─────────────────────────┴──────────┴──────┴─────────┴─────────┴───────────────┴──────┴──────────┴────────────┴─────────────┴────────────┴────────────┘
+```
+
+Now let's update the hyperparameters and run another experiment. There are
+several ways to do this with DVC:
+
+- Change the hyperparameter values directly in `params.yaml`
+- Update the values using the `--set-param` or the shorthand `-S` option on
+  `dvc exp run`
+- Queue multiple experiments with different values using the `--queue` option on
+  `dvc exp run`
+
+We'll do an example of each of these throughout the rest of this article.
+
+Let's start by updating the hyperparameter values in `params.yaml`. You should
+have these values in your file.
+
+```yaml
+lr: 0.009
+momentum: 0.017
+```
+
+Now run another experiment with `dvc exp run` and take a look at the metrics
+with `dvc exp show`. Your table should look something like this now. Since we're
+using checkpoints, note that we continue training additional epochs on top of
+your previous experiment.
```

Comment:
Do you think it's worth mentioning `--reset` here or saying that training from scratch will be shown below? Just a thought, but up to you.